Dockerfile

```diff
@@ -43,28 +47,31 @@ RUN echo " ... adding throttling support with ZMQ and CZMQ" \
          && rm -rf /tmp/zeromq* && rm -rf /tmp/czmq* \
          && rm -rf /var/cache/apk/*
 
-# openresty build
-ENV OPENRESTY_VERSION=1.13.6.1 \
-    PCRE_VERSION=8.37 \
-    TEST_NGINX_VERSION=0.24 \
-    _prefix=/usr/local \
+ENV _prefix=/usr/local \
     _exec_prefix=/usr/local \
     _localstatedir=/var \
     _sysconfdir=/etc \
     _sbindir=/usr/local/sbin
 
-RUN  echo " ... adding Openresty, NGINX, and PCRE" \
+# openresty build
+ENV OPENRESTY_VERSION 1.13.6.1
+ENV PCRE_VERSION 8.37
+RUN  echo " ... adding Openresty and PCRE" \
+     && OPENRESTY_SHA256=d1246e6cfa81098eea56fb88693e980d3e6b8752afae686fab271519b81d696b \
```

Comment:
for consistency, should we move `OPENRESTY_VERSION` here as well, instead of having it as an `ENV` ? B/c the SHA and the ENV must be changed together when updating a version.

Reply:
Yes I agree but `OPENRESTY_VERSION` is used in other layers.